nx.json

```diff
@@ -5,25 +5,26 @@
         "default",
         "{workspaceRoot}/.eslintrc.js"
       ],
-      "cache": true
+      "cache": false
```

Comment:
Do we want to log an issue to strip out nx entirely? Or are we keeping it around for the output?

Reply:
I don't think so. Since `lerna` is now owned by nx, the integration is pretty tight. I think it's easiest to likely just disable caching as it does some other things that are beneficial